### ISSUE TITLE
Textarea trims value

### ISSUE THREAD

**Issue by alanshaw** [choojs/choo#632]
### Expected behavior
```js
const text = "foo "
html`<textarea>${text}</textarea>`
```

output:

```html
<textarea>foo </textarea>
```

### Actual behavior

```html
<textarea>foo</textarea>
```

**Comment by goto-bus-stop** [comment]
Probably need to add an exception for textareas to the whitespace handling introduced in bel by this PR: https://github.com/shama/bel/pull/81

**Comment by goto-bus-stop** [comment]
fixed by https://github.com/shama/bel/pull/108, released in bel@5.1.7. do `npm update` to get it. thanks!
